### DISCUSSION TITLE
Adding new keywords like "int", "double", "string" ...

### DISCUSSION THREAD

**Discussion by adrien-thierry** [babel/babel#12126] [Q&A]
Hi,

We are developping NectarJS, a native JavaScript compiler (with c++ transpilation), and i'm using Babel to parse JS. 

We are able to use native c++ types with var types:

`var obj = {"one": 1, "two": "2"}
int i = obj.one;
string s = obj.two;`

What could be the best way to add "int", "double", "string" as keyworks like "var" in Babel ?

Thx

**Comment by JLHwung** [comment]
See https://github.com/babel/babel/discussions/11575

**Reply by adrien-thierry** [reply]
Thx @JLHwung, what could be the easiest way to add these keywords to the actual list (var, let) ?
